alembic/versions/614911daf883_add_seasonal_to_time_resolution_enum.py

```diff
@@ -0,0 +1,145 @@
+"""add 'seasonal' to time_resolution enum
+
+Revision ID: 614911daf883
+Revises: 
+Create Date: 2017-07-17 17:00:43.066818
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '614911daf883'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+connection = op.get_bind()
+dialect = connection.dialect.name
+
+
+# This is a *highly* customized migration script, barely related to the
+# autogenerated script at all. Two reasons:
+#
+# 1) There are a lot of differences between the (test, and presumably prod)
+#    database as it is now and the SQLAlchemy declaration. We ditch almost all
+#    those differences because we are frightened little creatures who don't
+#    want to wake monsters.
+# 2) Enum type. See below.
+#
+# Migration of an Enum type must be done differently depending on the database
+# type. ~sigh~
```

Comment:
Ah the old [law of leaky abstractions](https://www.joelonsoftware.com/2002/11/11/the-law-of-leaky-abstractions/). I.e. "we don't want to care about this, but we have to". Thanks for working this out.